assemblies/features/spring/src/main/feature/feature.xml

```diff
@@ -22,97 +22,97 @@
 
     <repository>mvn:org.apache.karaf.features/standard/${upstream.version}/xml/features</repository>
 
-    <!-- Spring 5.1.x support -->
+    <!-- Spring 5.3.x support -->
 
-    <feature name="spring" description="Spring 5.1.x support" version="${spring51.version}">
+    <feature name="spring" description="Spring 5.3.x support" version="${spring53.tesb.version}">
         <bundle dependency="true" start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aopalliance/${aopalliance.bundle.version}</bundle>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-core/${spring51.version}</bundle>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-expression/${spring51.version}</bundle>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-beans/${spring51.version}</bundle>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-aop/${spring51.version}</bundle>
+        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-core/${spring53.tesb.version}</bundle>
+        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-expression/${spring53.tesb.version}</bundle>
+        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-beans/${spring53.tesb.version}</bundle>
+        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-aop/${spring53.tesb.version}</bundle>
         <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-context/${spring-context.tesb.version}</bundle>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-context-support/${spring51.version}</bundle>
+        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-context-support/${spring53.tesb.version}</bundle>
     </feature>
 
-    <feature name="spring-aspects" description="Spring 5.1.x AOP support" version="${spring51.version}">
-        <feature version="[${spring51.version},5.2)">spring</feature>
-        <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-aspects/${spring51.version}</bundle>
+    <feature name="spring-aspects" description="Spring 5.3.x AOP support" version="${spring53.tesb.version}">
+        <feature version="[${spring53.tesb.version},5.4)">spring</feature>
```

Comment:
Why not reusing property `${spring53.tesb.version.range}` ?

Reply:
The version range is for all versions < Spring 6.0. Here it is about Spring aspects 5.3 which requires that the rest of Spring is also 5.3.x.

Reply:
done